src/uct/base/uct_iface.h

```diff
@@ -413,7 +413,7 @@ struct uct_iface_config {
 typedef struct uct_iface_mpool_config {
     unsigned          max_bufs;  /* Upper limit to number of buffers */
     unsigned          bufs_grow; /* How many buffers (approx.) are allocated every time */
-    double            grow_factor;/* growth for buffers' number compared to last time's */
+    double            grow_factor;/* Growth for buffers' number compared to last time's */
```

Comment:
Add space before comment

Reply:
`/* Increase each new allocated chunk by this factor */`

Reply:
fixed